GroceryStore/ProductDetailViewController.m

```diff
@@ -85,15 +93,47 @@ - (void)createURLRequestWithHTTPMethod:(NSString *)method {
     [self.URLRequest setValue:@"application/json" forHTTPHeaderField:@"Content-type"];
 }
 
-- (void)createAndHandleURLResponse:(NSURLResponse *)response withData:(NSData *)data {
+- (void)implementProductAdjustment
+{
+    //specify amount to add
+    NSDictionary *quantityDict = @{@"quantity" : @(self.stepperOutlet.value)};
+    
+    NSData *quantityToPass = [NSJSONSerialization
+                              dataWithJSONObject:quantityDict
```

Comment:
this is a valid way to have a multiline method call but it's often nicer to put the first two parts on the same line.  That way Xcode will align the following lines on the colons.